### PR TITLE
Task restrict access

### PR DESCRIPTION
**Pull request by rrrichard**
3 tests:
restrict access to admin page
you should be unable to access the page if session is not true
will redirect you to login if you havent logged in or session is not true

### CODE REVIEW

File: php/admin_add.php [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <?php
-
 require_once 'functions.php';
 require_once '../db/db_query.php';
 
-
```
Comment (milaferron) [inline php/admin_add.php:6]: remove whitespace

File: php/admin_page.php [hunk 2]
```diff
@@ -112,6 +118,6 @@
             </form>
         </div>
     </div>
-
```
Comment (milaferron) [inline php/admin_page.php:115]: remove whitespace

File: php/log_in_verify.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    $postUsername = $_POST['username'];
+    $test = usernameVerify($db, $postUsername, $postPassword);
+    if ($test == true || $_SESSION == true){
+        $_SESSION['valid'] = true;
+        header('Location: admin_page.php');
+    } else {
+        header('Location: log_in.php');
+    }
+}
+
```
Comment (milaferron) [inline php/log_in_verify.php]: remove whitespaces

File: php/log_in_verify.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?php
+session_start();
+require_once 'functions.php';
+require_once '../db/db_query.php';
+
+$db = getDbConnection();
+$postUsername = $_POST['username'];
```
Comment (milaferron) [inline php/log_in_verify.php]: put inside an if isset

**Review by milaferron — COMMENTED** [review]
you just have a few whitespaces to remove

**Review by rrrichard — COMMENTED** [review]
removed linebreaks as requested

**Pull request merged**